packages/react/src/components/InAppMessaging/CloseIconButton/CloseIconButton.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import { Link } from '../../../primitives';
+import { IconClose } from '../../../primitives/Icon/internal';
+
+import { CloseIconButtonProps } from './types';
+
+export function CloseIconButton({
+  className,
+  onClick,
+  style,
+  ...rest
+}: CloseIconButtonProps): JSX.Element {
+  return (
+    <Link className={className} onClick={onClick} style={style} {...rest}>
+      <IconClose size="1.5rem" />
```

Comment:
We can add an `aria-hidden="true"` here assuming we add the button label noted in the comment above.  Otherwise, screenreaders will read out redundant text; VoiceOver for instance adds to word "group" to the button name if it has an unlabeled image inside.